src/Dapr.Workflow/Dapr.Workflow.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- NuGet configuration -->
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <PackageId>Dapr.Workflow</PackageId>
+    <Title>Dapr Workflow Authoring SDK</Title>
+    <Description>Dapr Workflow SDK for building workflows as code with Dapr</Description>
+    <VersionPrefix>0.1.0</VersionPrefix>
+    <VersionSuffix>alpha</VersionSuffix>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DurableTask.Client.Grpc" Version="1.0.0-rc.1" />
+    <PackageReference Include="Microsoft.DurableTask.Worker.Grpc" Version="1.0.0-rc.1" />
+    <PackageReference Include="Dapr.AspNetCore" Version="1.8.0" />
```

Comment:
Nit: `1.9.0`?

Reply:
Yes, support for pluggable components only appears on 1.9+